doc/source/whatsnew/v2.0.0.rst

```diff
@@ -951,6 +951,10 @@ Indexing
 - Bug in :class:`BusinessHour` would cause creation of :class:`DatetimeIndex` to fail when no opening hour was included in the index (:issue:`49835`)
 -
 
+Copy on write
+^^^^^^^^^^^^^
+- Bug in :class:`DataFrame` constructor not tracking reference if called with another :class:`DataFrame` (:issue:`50499`)
```

Comment:
For the equivalent Series change I listed this in the "Copy-on-Write improvements" section instead of seeing it as a bug fix (it was a known gap left out of the initial implementation, and it also changes behaviour, so at this stage I wouldn't yet label it as a bug fix)